module/Activity/view/email/activity.phtml

```diff
@@ -0,0 +1,37 @@
+<?php $lang = 'en'; ?>
+<?= $this->translate('Hello,')?><br /><br />
```

Comment:
However, everyone that is dutch can read english emails. Not the other way around.

We can also put 'English text below' at the beginning of the email. And put an english text below the dutch text.